src/jit/compiler.cpp

```diff
@@ -14,6 +14,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #ifdef _MSC_VER
 #pragma hdrstop
 #endif // _MSC_VER
+#define __HORRID_HACK__
```

Comment:
Oh jeez that should never have made it into this branch, my apologies (too many balls in the air at the same time). I'll remove it immediately, of course - thanks for catching that!